locales/en-US/app.ftl

```diff
@@ -175,16 +175,13 @@ MarkerSettings--panel-search =
 
 ## These strings are used for the buttons at the top of the profile viewer.
 
-MenuButtons--index--metaInfo-button-uploaded-profile =
-    .label = Uploaded Profile
-
-MenuButtons--index--metaInfo-button-local-profile =
-    .label = Local Profile
+MenuButtons--index--metaInfo-button =
+    .label = Profile Info
 
 MenuButtons--index--full-view = Full View
 MenuButtons--index--cancel-upload = Cancel Upload
 MenuButtons--index--share-upload =
-    .label = Upload
+    .label = Upload Local Profile
```

Comment:
Note: this works while the project is not localized. Later, any change will require a new ID.

Reply:
Yes I'm aware of that :-) thanks for the notice though!